big_tests/tests/carboncopy_SUITE.erl

```diff
@@ -249,8 +248,7 @@ carbons_get_enabled(ClientOrClients) ->
 %%
 
 given_client_logs_out(Config, Client) ->
-    escalus_client:stop(Config, Client),
-    timer:sleep(300).
+    escalus_client:stop(Config, Client).
```

Comment:
Do we need this function right now? (as it's just one function call)

Reply:
Or maybe we should move the logic of waitng for "unavailable" presence here an then call it `given_client_logs_out`.